administrator/components/com_installer/Model/DatabaseModel.php

```diff
@@ -283,6 +288,150 @@ public function fix($cids = array())
 		}
 	}
 
+	/**
+	 * Export all the database via XML
+	 *
+	 * @return  boolean
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function export()
+	{
+		$db = $this->getDbo();
+
+		// Make sure the database supports exports before we get going
+		try
+		{
+			$exporter = $db->getExporter()->withStructure();
+		}
+		catch (UnsupportedAdapterException $e)
+		{
+			return false;
+		}
+
+		$tables = $db->getTableList();
+		$prefix = $db->getPrefix();
+
+		$zipFile = JPATH_ROOT . '/tmp/joomla_db.zip';
+		$zipArchive = (new Archive)->getAdapter('zip');
+
+		foreach ($tables as $table)
+		{
+			if (stristr($table, $prefix) != false)
```

Comment:
If the intention is to export only tables with current prefix, use this:
```suggestion
			if (strpos($table, $prefix)  === 0)
```